library.py

```diff
@@ -95,35 +97,36 @@ def add():
             if len(phone) != 10:
                 flash('Phone number must include area code. Please try agian.')
                 return redirect(url_for('admin'))
-            int(phone) # Confirms that phone is an integer
+            int(phone)  # Confirms that phone is an integer
         except:
             flash('Phone number must include area code. Please try agian.')
             return redirect(url_for('admin'))
     g.db = connect_db()
-    g.db.execute('INSERT INTO staff (username, password, f_name, l_name, phone) VALUES (?, ?, ?, ?, ?)', \
-                 [username,password,f_name, l_name, phone])
-    g.db.execute("INSERT INTO profile(username,bio) VALUES (?, ?)",[username,""])
+    g.db.execute('INSERT INTO staff (username, password, f_name, l_name, phone) VALUES (?, ?, ?, ?, ?)',
+                 [username, password, f_name, l_name, phone])
+    g.db.execute("INSERT INTO profile(username,bio) VALUES (?, ?)", [username, ""])
     g.db.commit()
     g.db.close()
     flash('New entry was successfully posted!')
     return redirect(url_for('admin'))
 
-@app.route('/edit-profile/<uname>',methods =['GET','POST'])
+
+@app.route('/edit-profile/<uname>', methods=['GET', 'POST'])
 @login_required
 def profile(uname):
     if session["logged_in_name"] != uname:
         return redirect(url_for('main'))
     else:
         g.db = connect_db()
-        cur = g.db.execute("SELECT bio FROM profile WHERE username=?",[uname])
+        cur = g.db.execute("SELECT bio FROM profile WHERE username=?", [uname])
         rows = cur.fetchall()
         try:
             bio = rows[0][0]
         except KeyError:
             flash("No profile found for user.")
             return redirect(url_for('main'))
-            
-        return render_template('profile.html',bio=bio)
+
+        return render_template('profile.html', bio=bio)
 
 if __name__ == '__main__':
-    app.run(debug=True)
\ No newline at end of file
+    app.run(debug=True, host='0.0.0.0')
```

Comment:
Added Host of 0.0.0.0 because I was looking at the code on a remote machine and couldn't see the web app because the default is 127.0.0.1.